### DISCUSSION TITLE
[Bug Report] Animation Export with Audio not work well as expected

### DISCUSSION THREAD

**Discussion by dabing1022** [keiwando/toonsquid#144] [General]
# app version:  1.7.8

# demo1
https://user-images.githubusercontent.com/1892303/216824216-d2435104-b493-4f79-9408-27d0a5e37f92.MP4

A simple stickman kick a ball and land. I move the "kick" audio clip to the moment of kicking the ball, and move the "land" audio clip to the moment of landing.

Here is the screenshot.

![IMG_0004](https://user-images.githubusercontent.com/1892303/216824923-f85614c2-7872-4a22-8ff0-45dc97e1afed.jpg)

The mp4 animation played with audios, but it doesn't play them at the right moments. It seems the audios are played at the frame begins.

# demo2

https://user-images.githubusercontent.com/1892303/216826771-cd39011d-5c5c-41c4-99b7-d205246599c0.MP4

audio clip 1 plays at 5th frame, audio clip 2 plays at 20th frame. But the exported animation plays at the beginning.

**Comment by keiwando** [comment]
Thanks for reporting this issue! I was able to reproduce it and have submitted the 1.7.9 update with a fix. It still needs to go through Apple review and will then be available on the App Store.

**Reply by dabing1022** [reply]
> Thanks for reporting this issue! I was able to reproduce it and have submitted the 1.7.9 update with a fix. It still needs to go through Apple review and will then be available on the App Store.

Cheers！
